lib/taskana-core/src/test/java/acceptance/task/CompleteTaskAccTest.java

```diff
@@ -1,42 +1,84 @@
 import pro.taskana.common.api.exceptions.NotAuthorizedException;
 import pro.taskana.common.api.exceptions.TaskanaException;
 import pro.taskana.common.internal.util.EnumUtil;
-import pro.taskana.common.test.security.JaasExtension;
 import pro.taskana.common.test.security.WithAccessId;
+import pro.taskana.task.api.TaskService;
 import pro.taskana.task.api.TaskState;
 import pro.taskana.task.api.exceptions.InvalidOwnerException;
 import pro.taskana.task.api.exceptions.InvalidStateException;
 import pro.taskana.task.api.exceptions.TaskNotFoundException;
+import pro.taskana.task.api.models.ObjectReference;
 import pro.taskana.task.api.models.Task;
-import pro.taskana.task.internal.models.TaskImpl;
-
-/** Acceptance tests for all claim and complete scenarios. */
-@ExtendWith(JaasExtension.class)
-class CompleteTaskAccTest extends AbstractAccTest {
+import pro.taskana.task.internal.builder.TaskBuilder;
+import pro.taskana.workbasket.api.WorkbasketPermission;
+import pro.taskana.workbasket.api.WorkbasketService;
+import pro.taskana.workbasket.api.models.WorkbasketSummary;
+import pro.taskana.workbasket.internal.builder.WorkbasketAccessItemBuilder;
+
+@TaskanaIntegrationTest
+class CompleteTaskAccTest {
+
+  @TaskanaInject TaskService taskService;
+  @TaskanaInject WorkbasketService workbasketService;
+  @TaskanaInject ClassificationService classificationService;
+  @TaskanaInject TaskanaEngine taskanaEngine;
+
+  ClassificationSummary defaultClassificationSummary;
+  WorkbasketSummary defaultWorkbasketSummary;
+  ObjectReference defaultObjectReference;
+
+  @WithAccessId(user = "businessadmin")
+  @BeforeAll
+  void setup() throws Exception {
+    defaultClassificationSummary =
+        defaultTestClassification().buildAndStoreAsSummary(classificationService);
+    defaultWorkbasketSummary = defaultTestWorkbasket().buildAndStoreAsSummary(workbasketService);
+
+    WorkbasketAccessItemBuilder.newWorkbasketAccessItem()
+        .workbasketId(defaultWorkbasketSummary.getId())
+        .accessId("user-1-1")
+        .permission(WorkbasketPermission.OPEN)
+        .permission(WorkbasketPermission.READ)
+        .permission(WorkbasketPermission.APPEND)
+        .buildAndStore(workbasketService);
+
+    defaultObjectReference = defaultTestObjectReference().build();
+  }
 
   @WithAccessId(user = "user-1-1")
   @Test
-  void testCompleteTask() throws Exception {
-    assertThat(taskService.getTask("TKI:000000000000000000000000000000000001").getState())
-        .isEqualTo(TaskState.CLAIMED);
-    Task completedTask = taskService.completeTask("TKI:000000000000000000000000000000000001");
+  void should_CompleteClaimedTask() throws Exception {
+    Task task = createTaskClaimedByUser();
+
+    assertThat(taskService.getTask(task.getId()).getState()).isEqualTo(TaskState.CLAIMED);
+
+    Task completedTask = taskService.completeTask(task.getId());
+
     assertThat(completedTask).isNotNull();
     assertThat(completedTask.getCompleted()).isNotNull();
```

Comment:
I know you didn't modify this, but maybe you're willing to make an adjustment here ^^

Instead of asserting modified is not created I'd personally assert that completed = modify != null != created.